Can't build the plugin with CE

I'm getting dependency resolution failures:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':google-cloud-tools-plugin:compileClasspath'.
> Could not find plugins/Groovy/lib/duplicates-groovy.jar (org.jetbrains.plugins:Groovy:2017.3).
  Searched in the following locations:
      file:/Users/eshaul/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2017.3/6b687670c9f987e74932316665ef9b58823a177/ideaIC-2017.3/org.jetbrains.plugins/Groovy/2017.3/plugins/Groovy/lib/duplicates-groovy-2017.3.jar
      file:/Users/eshaul/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2017.3/6b687670c9f987e74932316665ef9b58823a177/ideaIC-2017.3/plugins/Groovy/lib/duplicates-groovy.jar
      file:/Users/eshaul/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2017.3/34457b330cf276e55fcadd96d7c8be3410866b19/plugins/Groovy/lib/duplicates-groovy-2017.3-[classifier].jar
```

--- a/build.gradle
+++ b/build.gradle
@@ -27,7 +27,7 @@ buildscript {
 
 plugins {
     id 'net.researchgate.release' version '2.6.0'
-    id "org.jetbrains.intellij" version "0.3.1" apply false
+    id "org.jetbrains.intellij" version "0.3.4" apply false
     id "com.diffplug.gradle.spotless" version "3.6.0" apply false
 }
 

